### PR TITLE
Update sponsors for 2022 in PNG format.

### PR DESCRIPTION
**Pull request by lindenwells**
I tried to add an SVG but it was painful; some images mysteriously didn't appear. This is fine for now.

### CODE REVIEW

**Comment by andrewj-brown** [comment]
lgtm, although the cba and google logos look almost comically small tbh

**Review by ndrwbrwn — APPROVED** [review]
lgtm - it's centered for me, check with tom what was happening to his setup - although the cba and google logos look almost comically small tbh

**Comment by TomCranitch** [comment]
I'm not too sure, but im getting this issue where the image is crazy big on firefox and chrome

File: layouts/partials/sponsor-banner.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
             </div>
 
             <!-- object tag enables links inside the SVG image -->
-            <object class="sponsor-img" type="image/svg+xml" data="{{ "/img/sponsors/all_sponsors.svg" | relURL }}">
+            <object class="sponsor-img" type="image/png" data="{{ "/img/sponsors/all_sponsors_2022.png" | relURL }}">
```
Comment (TomCranitch) [inline layouts/partials/sponsor-banner.html:10]: Also I just looked over this properly but is there any reason not to use an `img` tag

Comment (andrewj-brown) [reply layouts/partials/sponsor-banner.html:10]: currently no but based on that html comment it'll be needed once we get a nice, modular SVG version going

**Comment by andrewj-brown** [comment]
> I'm not too sure, but im getting this issue where the image is crazy big on firefox and chrome

I'm on firefox, no troubles. Strange that it's happening to your firefox and not mine.

File: layouts/partials/sponsor-banner.html (hunk 1 above)
Comment (lindenwells) [reply layouts/partials/sponsor-banner.html:10]: Yeah it was just me being lazy and changing "xml+svg" to "png" or something.
Apologies I sorta half-arsed that after failing for a few hours

**Pull request merged**